packages/table/src/locator.ts

```diff
@@ -76,6 +78,27 @@ export class Locator implements ILocator {
         return max;
     }
 
+    public getTallestVisibleCellInColumn(columnIndex: number): number {
+        const cells = this.tableElement.querySelectorAll(`.${Classes.columnCellIndexClass(columnIndex)}`);
+        let max = 0;
+        for (let i = 0; i < cells.length; i++) {
+            const cellValue = (cells.item(i) as any).query(`.${Classes.TABLE_TRUNCATED_VALUE}`);
+            const cellTruncatedFormatText = (cells.item(i) as any).query(`.${Classes.TABLE_TRUNCATED_FORMAT_TEXT}`);
+            let height = 0;
+            if (cellValue != null) {
+                height = cellValue.scrollHeight;
+            } else if (cellTruncatedFormatText != null) {
+                height = cellTruncatedFormatText.scrollHeight;
+            } else {
+                height = (cells.item(i) as any).query(`.${Classes.TABLE_TRUNCATED_TEXT}`).scrollHeight;
+            }
+            if (max < height) {
```

Comment:
nit: I'd phrase this as `height > max`, because I normally prefer the current thing to be compared to the existing thing, left-to-right.